webpack/components/PreupgradeReportsList/components/__tests__/EmptyInfoItem.test.js

```diff
@@ -1,13 +1,14 @@
 import { testComponentSnapshotsWithFixtures } from '@theforeman/test';
 
 import EmptyInfoItem from '../EmptyInfoItem';
+import reportEntry from './fixture';
 
-const fixtures = {
-  'should render': {
-    entry: { title: 'no title', severity: 'low', id: 5, preupgradeReportId: 7 },
-    attr: 'title',
-  },
-};
+// const fixtures = {
+//   'should render': {
+//     entry: { title: 'no title', severity: 'low', id: 5, preupgradeReportId: 7 },
+//     attr: 'title',
+//   },
+// };
 
 describe('EmptyInfoItem', () =>
-  testComponentSnapshotsWithFixtures(EmptyInfoItem, fixtures));
+  testComponentSnapshotsWithFixtures(EmptyInfoItem, reportEntry));
```

Comment:
The props sturucture for `EmptyInfoItem` has not changed, so you can keep the fixture as is. You can also replace the current `entry` in `fixtures` with the `reportEntry` if you need to.